regress/unittests/test_helper/test_helper.c

```diff
@@ -218,7 +218,8 @@ void
 test_start(const char *n)
 {
 	assert(active_test_name == NULL);
```

Comment:
revert changes to regress/modpipe.c above

Why is this change (in test_helper.c) needed? This is base code, if there is an issue here - we also need to report and get it fixed in upstream branch.

Reply:
@manojampalam assertions aren't compiled in release mode.  As a result, active_test_name is NULL when the test name is printed.  Luckily this just results in the string "(null)" since printf is smart enough to do a null pointer check.  However, it's not very useful to see which test actually failed.